src/main/java/Duke/Event.java

```diff
@@ -0,0 +1,28 @@
+package Duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task that will occur on a particular date
+ */
+

```

Comment:
Great to see javadocs for public classes and methods! You might have missed this out accidentally, but remember that there should not be any blank line between your header comment and your class/method

Reply:
You're right, thank you for pointing out